### ISSUE TITLE
Gradle build error

### ISSUE THREAD

**Issue by Nagaraj668** [lzyzsd/CircleProgress#74]
I'm getting Gradle build error,

Failed to resolve: compile 'com.github.lzyzsd:circleprogress:1.1.2'

**Comment by xperiafan13-rom** [comment]
same error

**Comment by youssefmanouare** [comment]
same here

**Comment by albertoha94** [comment]
Seems like the lib is dead as there's no way to use it with gradle.

**Comment by youssefmanouare** [comment]
@albertoha94  try to integrate classes and attr in your project by coping them or integrate the librairie as a module , all this instead of using gradle . it's working for me

**Comment by albertoha94** [comment]
I wanted to update my answer.
Try this instead:
    compile 'com.github.lzyzsd:circleprogress:+'
Worked for me.

**Comment by youssefmanouare** [comment]
@albertoha94  yes it's working , i tried  it but **donut_inner_drawable** ,**donut_show_text** will not work

**Comment by albertoha94** [comment]
@youssefmanouare Did you added the styleables in a resource file?

**Comment by youssefmanouare** [comment]
@albertoha94  yes add them to your attrs.xml ressouce file

**Comment by albertoha94** [comment]
Sorry but i don't know what could be the problem. I'm using circle progress only and it seems good so far.

**Comment by youssefmanouare** [comment]
@albertoha94 yes it's worrking with the gradle that you gave below but the attruibutes  donut_inner_drawable ,donut_show_text will not work , so i integrate all classes in my project and it's working fine without gradle

**Comment by albertoha94** [comment]
Oh I see. That's another solution then.

**Comment by youssefmanouare** [comment]
@albertoha94  yes , an other one is add the librarie as a module of your project , it's will work fine also.

**Comment by odyodyodys** [comment]
# Solution
use `compile 'com.github.lzyzsd:circleprogress:v1.1.1'`
see #76 for more

**Comment by AndrewParkhomov** [comment]
repositories {
            jcenter()
            mavenCentral()
            maven { url "https://jitpack.io" }
        }
work for me
